ethics/EXAMPLES.md

```diff
@@ -0,0 +1,48 @@
+
+# Examples of situations for consideration
+
+## Ability to be forgotten
+
+### Deletion is delete not flagging
+
+When an account is deleted it must be truly deleted, not just flagged for future delete. Note
+that some functionality like the recycle bin, we must keep the account details, but a recycle
+bin purge does truly delete the account.
+
+## Self determination and autonomy
+
+### Self name change
+
+People should be able to change their own name at anytime. Consider divorce, leaving abusive partners
+or other personal decisions around why a name change is relevant.
+
+This is why names are self-service writeable at any time.
+
+## Free from harmful discrimination of any kind
+
+### Cultural and Social awareness of name formats
+
+All name fields should be case sensitive utf8 with no max or min length limit. This is
```

Comment:
naive question, do you _need_ more than one name field (like, is there a benefit to the first name / lastname / whatever arrangement vs. just having one name field)? just that some people that will want it?

and, on the basis that all these fields are customisable anyway (afaik from talking to you about this, might be incorrect), is there a way that you could recommend but also gently influence this (with a default configuration or something)?

Reply:
Ahhh name fields are displayName/legalName. They have different privacy settings by default in the project. But I should make taht clearer (there is plenty of documentation about why firstname/lastname is not usable).

Reply:
One of my biggest frustrations with software is the name/preferredName structure, which guides people into always using the wrong one. I definitely like your approach.